controller/upload_file.rst

```diff
@@ -442,10 +442,17 @@ controller.
             public function postLoad(LifecycleEventArgs $args)
             {
                 $entity = $args->getEntity();
-
-                $fileName = $entity->getBrochure();
-
-                $entity->setBrochure(new File($this->targetPath.'/'.$fileName));
+                
+                // Only for tests purpose and proper object validation
```

Comment:
Checking for the right object below indeed is needed. However, this comment looks confusing to me and I would just remove it.